### DISCUSSION TITLE
Move zod-form-data out of remix-validated-form?

### DISCUSSION THREAD

**Discussion by einarpersson** [airjp73/rvf#237] [General]
I started using [zod-form-data](https://www.npmjs.com/package/zod-form-data) since that was exactly what I needed in my SvelteKit project. But the repository is named [remix-validated-form](https://github.com/airjp73/remix-validated-form) which makes me confused and unsure about what the direction/intent is.

If `zod-form-data` is meant to be a framework-agnostic helper then maybe it should be in another repository? It is a great helper that has value outside remix ecosystem.

If not, consider linking directly to https://github.com/airjp73/remix-validated-form/tree/main/packages/zod-form-data from https://www.npmjs.com/package/zod-form-data instead of to remix-validated-form

**Comment by airjp73** [comment]
I've definitely considered that, but I think the overall maintenance burden is lower having it all in one repo for now. Good idea about updating the repo link though! I would definitely take a PR updating that if you had the time.
